MaxText/layers/quantizations.py

```diff
@@ -265,6 +264,8 @@ def _get_aqt_key_paths(aqt_vars):
       if "AqtDotGeneral" in d.key:
         pruned_keys.append(jax.tree_util.DictKey(key="kernel"))
         break
+      elif "AqtEinsum" in d.key:
```

Comment:
Still need this? i.e. for inference only?

Reply:
This code is used by MaxEngine (inferenece only) to convert checkpoint from bf16 to int8. Still needed.